### ISSUE TITLE
@IsUrl options not works

### ISSUE THREAD

**Issue by sanex3339** [pleerock/class-validator#49]
It seems what `@IsURL` options not works at all.

```javascript
   /**
     * @type {string}
     */
    @IsString()
    @ValidateIf((options: IOptions) => Boolean(options.sourceMapBaseUrl))
    @IsUrl({
        require_protocol: true,
        require_valid_protocol: true
    })
    public readonly sourceMapBaseUrl: string;
```

Input string `google.ru` -> validator does not trows any errors.

**Comment by sanex3339** [comment]
ping

**Comment by pleerock** [comment]
as you know `IsUrl` just delegates its work to https://github.com/chriso/validator.js isUrl. Can you check if what you try to achieve works with original function? e.g. `require("validator").isURL(value, options)`

**Comment by sanex3339** [comment]
https://runkit.com/58259b2fcfbc960014b4591f/582ab7ae430fac0014ffc6d4
```javascript
    require("validator").isURL('google.ru', {
        require_protocol: true,
        require_valid_protocol: true
    })
```
returns `false`


Looks like `class-validator` not passes options for `url` validator.

**Comment by pleerock** [comment]
try fix in `0.6.5`

**Comment by sanex3339** [comment]
ts error with 0.6.5
` Cannot find type definition file for 'es6-shim'. `

**Comment by pleerock** [comment]
damn this is problem with new `@types` system... I'll fix it, but despite the compiler errors, is `isUrl` working now?

**Comment by sanex3339** [comment]
looks like now it works

**Comment by pleerock** [comment]
fixed your issue in `0.6.6`

**Comment by sanex3339** [comment]
Amazing, thank you!
